lib/network/Images.js

```diff
@@ -1,4 +1,4 @@
-import CachedImage from './CachedImage';
+var CachedImage = require('./CachedImage').default;
```

Comment:
`CachedImage` is an ES6 module because it is exported with ES6 export syntax.
`Images` is also an ES6 module.  `export default Images;` at the end.

So it's again mixing styles because it's importing `CachedImage` with a cjs syntax inside an ES6 module.